Web/Edubase.Web.UI/Views/Glossary/Index.cshtml

```diff
@@ -49,7 +49,7 @@
                             <div class="govuk-grid-row">
                                 <div class="govuk-grid-column-two-thirds govuk-!-padding-bottom-2">
                                     <h2 class="govuk-heading-s">@item.Title</h2>
-                                    <div class="gias-user-content">@Html.Raw(item.Content)</div>
+                                    <div class="gias-user-content">@Html.Raw(item.Content.Replace("\n", "<br/>"))</div>
```

Comment:
Are newlines within `item.Content` guaranteed to only ever be `\n`?

Consider this more general alternative which matches on any of CRLF / CR / LF (generally these are Windows/Mac/Linux linebreaks):

```csharp
 <div class="gias-user-content">@Html.Raw(Regex.Replace(item.Content, @"\r\n|\r|\n", "<br/>"))</div>

```

https://dotnetfiddle.net/uk1oaH

Reply:
I agree with Roger.

Reply:
@RogerHowellDfE Done, thanks for the suggestion

@anthony9853